packages/fuels/src/cli/commands/deploy/proxy/contract/fuel-toolchain.toml

```
@@ -0,0 +1,6 @@
+[toolchain]
+channel = "testnet-2024-09-06"
+
+[components]
+forc = "0.63.5"
```

```suggestion
forc = "0.63.6"
```

Should we be using latest [`0.63.6`](https://github.com/FuelLabs/sway/releases/tag/v0.63.6)